examples/chart/teleport/README.md

```diff
@@ -60,6 +60,19 @@ $ kubectl create configmap ca-certs --from-file=ca.pem
 $ helm upgrade --install teleport ./
 ```
 
+## Downloading the chart from the Gravitational repo
```

Comment:
Maybe as a separate PR, we should add this info at https://gravitational.com/teleport/docs/kubernetes_ssh/